python/cudf_kafka/pyproject.toml

```diff
@@ -86,4 +86,5 @@ requires = [
     "cmake>=3.30.4",
     "cython>=3.0.3,<3.1.0a0",
     "ninja",
+    "numba-cuda>=0.11.0",
```

Comment:
We probably don't want to add this as a build dependency for `cudf-kafka`. You may need to specify that build dependency in a more specific list for just cudf (or pylibcudf? unsure).

Reply:
This should be resolved in 153976d4a8d2af7e44f5890101124da7c0f02c6d.